src/metabase/analytics/stats.clj

```diff
@@ -258,7 +258,7 @@
   "Get metrics based on pulses
   TODO: characterize by non-user account emails, # emails"
   []
-  (let [pulse-conditions {:left-join [Pulse [:= :pulse.id :pulse_id]], :where [:= :pulse.alert_condition nil]}]
+  (let [pulse-conditions {:left-join [:pulse [:= :pulse.id :pulse_id]], :where [:= :pulse.alert_condition nil]}]
```

Comment:
Honey SQL 2 doesn't understand Toucan 1 models (and can't be made to); Toucan 2 models are simple keywords, so this query will break when we switch over to Toucan 2. I replaced the Toucan 1 model with using the table name directly.